test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/BookKeeperAdapter.java

```diff
@@ -248,7 +248,7 @@ static Process startBookKeeperOutOfProcess(TestConfig config, String logId) thro
                 .stdOut(ProcessBuilder.Redirect.to(new File(config.getComponentOutLogPath("bk", 0))))
                 .stdErr(ProcessBuilder.Redirect.to(new File(config.getComponentErrLogPath("bk", 0))))
                 .start();
-        ZooKeeperServiceRunner.waitForServerUp(config.getZkPort());
+        ZooKeeperServiceRunner.waitForServerUp(config.getZkPort(), false, "", "", "", "");
```

Comment:
Should we have a separate signature for `waitForServerUp` when secure ZK is false so that we don't have to pass a bunch of empty strings?

Reply:
Fixed. Created a default implementation.